src/index.html

```diff
@@ -93,13 +52,12 @@
   </biggive-totalizer>
 
 
-
-
     <biggive-hero-image
         colour-scheme="primary"
-        main-image="https://media.istockphoto.com/vectors/childish-seamless-dotted-pattern-with-colorful-doodle-letters-fun-vector-id1208462693"
-        main-title="Christmas Challenge 2022"
+        main-title="Christmas Challenge 2024"
         main-title-colour="primary"
+        main-image="http://biggive.thomas-paterson.co.uk/wp-content/uploads/2023/02/Grid.png"
+        main-image-shape="rectangle"
```

Comment:
the image link used before didn't seem to be working. Ideally we wouldn't depend on this site but its already used many times in index.html, we can replace all of them together at some point.